### PR TITLE
Add missing import / requirement

### PR DESCRIPTION
**Pull request by jeffwidman**
These are needed for the imports / requirements.

### CODE REVIEW

File: examples/django_demo/generic_foreignkey/factories.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django.contrib.auth.models import Group, User
 from django.contrib.contenttypes.models import ContentType
 
+import factory
```
Comment (kingbuzzman) [inline examples/django_demo/generic_foreignkey/factories.py:4]: Why is this needed?

Comment (jeffwidman) [reply examples/django_demo/generic_foreignkey/factories.py:4]: Several lines in this file use `factory.X` but the `factory` import itself was missing... eg:

https://github.com/FactoryBoy/factory_boy/blob/6b9beb9a4198b7bd01c61f3fc8a85859f2d361e7/examples/django_demo/generic_foreignkey/factories.py#L24

Comment (kingbuzzman) [reply examples/django_demo/generic_foreignkey/factories.py:4]: Oh I see, yes good catch!

**Review by kingbuzzman — APPROVED** [review]

File: examples/django_demo/generic_foreignkey/factories.py (hunk 1 above)
Comment (jeffwidman) [reply examples/django_demo/generic_foreignkey/factories.py:4]: Oh interesting, I either never realized that or long ago forgot that!
